tests/test_cli_socket_operation_deadline.py

```
@@ -0,0 +1,191 @@
+    conn.close()
+
+
+def run_cli(cli_path: str, socket_path: str, timeout: float = 3.0) -> RunResult:
+    env = dict(os.environ)
+    env["CMUX_SOCKET_PATH"] = socket_path
+    env["CMUX_SOCKET"] = socket_path
+    env["CMUXTERM_CLI_RESPONSE_TIMEOUT_SEC"] = "0.2"
+    env["CMUX_CLI_SENTRY_DISABLED"] = "1"
+    env["CMUX_CLAUDE_HOOK_SENTRY_DISABLED"] = "1"
+    started = time.monotonic()
+    proc = subprocess.run(
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)